src/oss/confluent.sh

```diff
@@ -1312,16 +1312,18 @@ extract_json_config() {
 
 produce_command() {
     local topicname="${1}"
+    shift
     local args="$@"
 
     if [[ "x${topicname}" == "x" ]]; then
-        echo "Missing required topic name argument in '${command_name} produce <topicname>'"
+        echo -e "Error: Missing required topic name argument in '${command_name} produce <topicname>'\n"
```

Comment:
Why `echo -e` ? Doesn't seem to be needed.
It should echo to stderr though (`echo "Error: Missing... " 1>&2`)

Reply:
`-e` is to add a new line (note the `\n` at the end) to separate it from the usage that gets printed right afterwards